### ISSUE TITLE
When using jQuery .remove() needs to be .detach() in the view

### ISSUE THREAD

**Issue by georgehenderson** [mutualmobile/lavaca#27]
When using jQuery instead of Zepto .remove() removes the event handlers. It can be resolved by substituting .detach() for .remove() in the view.
